app/controllers/checkouts_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CheckoutsController < ApplicationController
+  before_action :set_trip_join_request, only: %i[ edit update payment_receipt ]
+
+  def index
+  end
+
+  def new
+    @trip_join_request = TripJoinRequest.new
+  end
+
+  def edit
+  end
+
+  def create
+    trip_join_request = TripJoinRequest.new
+    @checkout = trip_join_request.transfer_receipt(checkout_params)
```

Comment:
just a style comment. do you really need to do this in two lines? isn't enough
`@checkout = TripJoinRequest.new.transfer_receipt(checkout_params)`